src/presentation/http/router/note.ts

```diff
@@ -84,7 +84,6 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
       accessRights: {
         canEdit: boolean,
       },
-      tools: EditorTool[],
```

Comment:
why tools has been removed? we need them to render a note

Reply:
because now tools are inside note, we don't need one more field on one level with note

Reply:
you mean that we should return to web full editor tools? like note { ... ~~tools~~}